src/commands/acronym.ts

```diff
@@ -0,0 +1,30 @@
+import { Command, flags } from "@oclif/command"
+import { findAcronym } from "../lib/acronyms"
+
+export default class Acronym extends Command {
+  static aliases = ["tla", "wtf"]
+
+  static description = "Explain an acronym heard at Artsy"
+
+  static examples = [
+    `$ artsy acronym aov
+AOV: Average Order Value`,
+  ]
+
+  static flags = {
+    help: flags.help({ char: "h" }),
+  }
+
+  static args = [{ name: "query" }]
```

Comment:
We might consider adding `required: true` to this arg, otherwise you get an error if it's not supplied:

```
$ bin/run acronym                 
TypeError: Cannot read property 'toLowerCase' of undefined
    at ~/code/artsy/artsy-cli/src/lib/acronyms/index.ts:5:47
    at Array.find (<anonymous>)
    at Object.exports.findAcronym (~/code/artsy/artsy-cli/src/lib/acronyms/index.ts:4:44)
    at Acronym.run (~/code/artsy/artsy-cli/src/commands/acronym.ts:22:20)
    at Acronym._run (~/code/artsy/artsy-cli/node_modules/@oclif/command/lib/command.js:44:31)
```

vs

```
$ bin/run acronym
 ›   Error: Missing 1 required arg:
 ›   query
 ›   See more help with --help
```

Might also rename the arg `acronym` so that the input is a bit clearer?